src/stavenote.ts

```diff
@@ -152,7 +152,7 @@ export class StaveNote extends StemmableNote {
     const draw = [false, false, false];
 
     for (let i = 0; i < notesList.length; i++) {
-      draw[i] = notesList[i].note.renderOptions.draw == false ? false : true;
+      draw[i] = notesList[i].note.renderOptions.draw === false ? false : true;
```

Comment:
This is a totally weird line of code :-). I guess it is because .draw can be true or false or undefined.

```
  public renderOptions: {
    drawStemThroughStave?: boolean;
    draw?: boolean;
```

Can we rewrite it as:

```ts
// If .draw is true or undefined, we set draw[i] = true
draw[i] = notesList[i].note.renderOptions.draw !== false;
```

Reply:
done